src/main/java/mate/controller/IndexController.java

```diff
@@ -1,9 +1,30 @@
 package mate.controller;
 
+import java.io.IOException;
+import java.time.LocalDateTime;
+import javax.servlet.ServletException;
 import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
 import mate.dao.MyCoolResource;
 
 public class IndexController extends HttpServlet {
     private MyCoolResource myResource;
-    //TODO: implement
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        myResource.write(LocalDateTime.now().toString());
+        req.getRequestDispatcher("/WEB-INF/views/index.jsp").forward(req, resp);
+    }
+
+    @Override
+    public void init() {
+        myResource = MyCoolResource.openResource();
+    }
```

Comment:
I think the init() method should be the first, because we need to initialize the object myResource before the method write()